Add Digistump logo as background for the content div

kh90909 opened the pull request.

kh90909 committed:
Add Digistump logo as background for the content div

emcniece commented:
Looks great! What a way to add branding... very cool.

I have 2 screens that differ in brightness - the darker one is ok, but the text on the bright one interferes with the logo, it's a bit hard to read with the graphic behind it. Adding this graphic as a background image on a standalone `div` is the perfect implementation, as we can change the opacity of a single element without affecting surrounding children. 

Is changing the opacity right now a must-have for alpha/MVP? I'm fine with leaving it as-is, just wanted to note my own preference for lower opacity.

kh90909 commented:
Opacity is hard coded in the image at the moment due to originally planning
to make it the background of the #content div, but had to abort that plan
because the height of that div grows with its contents which throws off the
positioning.

Makes sense to go back to the original image and implement the opacity on
the div as you suggested.

Happy to make it more transparent. Better to have it be too transparent for
some than too opaque for anyone.
On May 11, 2016 1:44 AM, "Eric McNiece" notifications@github.com wrote:

> Looks great! What a way to add branding... very cool.
> 
> I have 2 screens that differ in brightness - the darker one is ok, but the
> text on the bright one interferes with the logo, it's a bit hard to read
> with the graphic behind it. Adding this graphic as a background image on a
> standalone div is the perfect implementation, as we can change the
> opacity of a single element without affecting surrounding children.
> 
> Is changing the opacity right now a must-have for alpha/MVP? I'm fine with
> leaving it as-is, just wanted to note my own preference for lower opacity.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/kh90909/OakTerm/pull/74#issuecomment-218367277

emcniece commented:
Alright. For reference, my bright screen works well with a `#content-background` opacity of `0.5` and the logo is still present for the dark screen. What does that look like for you?

kh90909 commented:
I'll take a look at this tomorrow. I've turned off my computer for the
night.

FYI in the PNG committed, I reduced the opacity to 20%, so 0.5 would equal
0.1 for the original image.
On May 11, 2016 2:03 AM, "Eric McNiece" notifications@github.com wrote:

> Alright. For reference, my bright screen works well with a
> #content-background opacity of 0.5 and the logo is still present for the
> dark screen. What does that look like for you?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/kh90909/OakTerm/pull/74#issuecomment-218369700

kh90909 committed:
Control background image opacity via css

Previously, it was hardcoded in the image. Also made the image more
transparent to avoid interfering with the foreground content on certain
monitors.

kh90909 commented:
This opacity level looks fine for me too, so I've updated the PR. I also went back to the original image so that the opacity is entirely controlled by the CSS.
